docs/articles/tutorial/3-using-workflows.mdx

```diff
@@ -0,0 +1,91 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Using workflows
+
+## Creating your first test workflow
```

Comment:
I'm not sure if that makes sense - considering that we throwing here some random TestWorkflow example with some comments, maybe it's better to just show Users how to use "Wizard" or "Examples" option in the UI?